lusidtools/cocoon/seed_sample_data.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+from lusidtools.cocoon.cocoon import load_from_data_frame
+from lusidtools.cocoon.utilities import load_json_file
+import logging
+
+logger = logging.getLogger()
+
+mappings = dict(load_json_file("config/seed_sample_data.json"))
+
+def seed_sample_data(api_factory, file_types, scope: str, transaction_source, mappings: dict = mappings):
+
+    """
+    Description:
+    This function allows users to seed their LUSID environment with some core data (e.g. instruments,
+    portfolios, and transaction) from one file.
+
+    :param lusid.utilities.ApiClientFactory api_factory: The api factory to use.
+    :param list[str] file_types: A list of the file_types for upload.
+    :param str scope: The scope of the transaction portfolio.
+    :param str transaction_source: a CSV file of data.
```

Comment:
should this be something more general like `source_file` i.e. could be a position file
is it restricted to being a csv or should it also support Excel?

Reply:
The file is passed to pandas.read_csv so we should pass a CSV.

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html

I can add a condition for read_excel also?

Reply:
I think we do need to support Excel given the sample files we've seen to date